docs/reference/components/kafka.cue

```diff
@@ -58,6 +58,17 @@ components: _kafka: {
 				examples: ["10.14.22.123:9092,10.14.23.332:9092"]
 			}
 		}
+		statistics_interval_ms: {
+			common:      false
+			description: "Interval in which statistics about Kafka should be polled and emitted into the `internal_metrics` stream.\n"
```

Comment:
Just wondering why the entry above this `bootstrap_servers.description` doesn't have a `\n` at the end, but the one below `librdkafka_options.description`. What's the correct variant?